mongo-helm/values.yaml

```diff
@@ -0,0 +1,17 @@
+appName: mongodb
+
+mongoexpress:
+  appName: mongo-express
+  imageName: mongo-express
+  imageTag: latest
+data:
+  database_url: mongodb-service
+
+mongo:
+  imageName: mongo
+  imageTag: latest
+
+service:
+  name: mongo-express-service
+  nodePort: 30000
+  
```

Comment:
I do not see the application here. Did you only package the database in helm chart?

Reply:
Missed that, fixed now.